deid/cms_mapping_init.sql

```diff
@@ -17,7 +17,7 @@ create sequence bene_id_deid_seq
 
 -- De-identified msis_id (1:1 msis_id to sequence number mapping)
 create sequence msis_id_deid_seq
-  start with 1
+  start with &&msis_id_deid_seq_start

```

Comment:
Where is `msis_id_deid_seq_start` defined? The README documents some parameters; should this be documented there?

Reply:
made inline comment as well as added in README